.gitignore

```diff
@@ -12,4 +12,5 @@ project/target
 block-explorer-handler*.jar
 block-explorer-api*.jar
 *.pem
-.terraform.lock.hcl
\ No newline at end of file
+.terraform.lock.hcl
+ec2-cluster/modules/node/data/
```

Comment:
should it really be ignored? if so, everyone would have to create own keys?

Reply:
You must pass `p12` files with the corresponding `id` as terraform input. We can either commit the keys or keep the dir ignored. Eventually, pass path to keys as input so we do not need to ignore and create any special directories for that 🤔